### DISCUSSION TITLE
Discogs Plugin Is Not Working For Me. Please Help

### DISCUSSION THREAD

**Discussion by bkresnick** [beetbox/beets#5273] [Q&A] [answered]
Hello. After updating to the latest version. Now import does not get discogs data for autotagging). Below is output from my testing with “Who Killed Sgt. Pepper?”

The discogs plugin is installed / configured as per documentation. I tried re-authenticating my Discogs connection. That worked but did not change the import functionality.

Thanks for an otherwise stellar piece of software.

MacBook-Air:~ user$ beet --version
beets version 2.0.0
Python version 3.10.4
plugins: discogs, edit, export, fetchart, ftintitle, inline, the, types

Macbook-Air:~ User$ beet -vv import -Ct /Volumes/Data/Music2/AntnNwcmb2010_auk020
user configuration: /Users/User/.config/beets/config.yaml
data directory: /Users/User/.config/beets
plugin paths: 
fetchart: google: Disabling art source due to missing key
fetchart: lastfm: Disabling art source due to missing key
inline: adding item field catalognum_adj
inline: adding album field albumartist_adj
inline: adding album field initials
inline: adding album field longAA
inline: adding album field shortAA
inline: adding album field catalognum_adj
inline: adding album field andAA
inline: adding album field and_adj
Sending event: pluginload
library database: /Users/User/.config/beets/library.db
library directory: /Users/User/Downloads/MusicTest
Sending event: library_opened
Sending event: import_begin
Sending event: import_task_created
/Volumes/Data/Music2/AntnNwcmb2010_auk020
0 of 13 items replaced
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
fetchart: trying source filesystem for album Anton Newcombe - Who Killed Sgt. Pepper?
fetchart: using well-named art file auk020_art.jpg
fetchart: using local image /Volumes/Data/Music2/AntnNwcmb2010_auk020/auk020_art.jpg
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: import_task_files
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: database_change
Sending event: album_imported
Sending event: import
Sending event: cli_exit

**Comment by DjSlash** [comment]
Could you provide the normal output, without the verbose stuff?

**Comment by bkresnick** [comment]
There is none. Beets imports the album. No options appear despite timid
mode.

**Reply by DjSlash** [reply]
Huh. Could you please provide your config?

**Comment by bkresnick** [comment]
ignore:
  FLAC
  Flac
  flac
  WAV
  Wav
  wav
  96 KHz

asciify_paths:
  default: yes

replace:
    '[&+]': 'And'
    '[()!#.?,~]': ''
    '[\\/]': _
    '^\.': _
    '[\x00-\x1f]': _
    '[<>:"\?\*\|]': _
    '\.$': _
    '\s+$': ''
    '^\s+': ''
    '^-': _
  #wondering about $, %, @, ^, =, :;<>|{}

plugins: discogs the inline fetchart ftintitle edit export types play
#bandcamp = beetcamp
#importinspect, copyartifacts > unmaintained / recent py issues
#extrafiles replaces copyartifacts? But bugs out when installed

#rewrite: >> scrapping this bc sublabel names might overlap artist names
 #albumartist Anything Goes: Rollover Milano Records

export:
    json:
        formatting:
            ensure_ascii: false
            indent: 4
            separators: [',' , ': ']
            sort_keys: true
    csv:
        formatting:
            delimiter: ','
            dialect: excel


play:
  command: /Applications/VLC.app/Contents/MacOS/VLC --no-playlist-autostart
  warning_threshold: 100
  raw: yes

extrafiles:
    patterns:
        all: '*.*'

importinspect:
  on_apply: yes
  timid: yes
  ignored: []
  ignored_new:
[acoustid_fingerprint,acoustid_id,added,album,album_id,albumartist_credit,albumartist_sort,albumdisambig,albumstatus,albumtotal,albumtypes,arranger,artist,artist_credit,artist_sort,artpath,asin,bitdepth,bitrate,bpm,catalognum_adj,channels,comments,composer,composer_sort,country,day,disc,discogs_albumid,discogs_artistid,discogs_labelid,disctitle,disctotal,encoder,filesize,format,id,initial_key,initials,isrc,language,length,longAA,lyricist,lyrics,mb_albumartistid,mb_albumid,mb_artistid,mb_releasegroupid,mb_releasetrackid,mb_trackid,mb_workid,month,mtime,original_day,original_month,original_year,path,r128_album_gain,r128_track_gain,releasegroupdisambig,rg_album_gain,rg_album_peak,rg_track_gain,rg_track_peak,samplerate,script,shortAA,singleton,style,title,track,trackdisambig,tracktotal,work_disambig,year]

discogs:
  source_weight: 0.0
  index_tracks: yes
  User_token: fPVEiOeWSwrohbGXDnfbmlNdWfwVqJzwACNhqURc

bandcamp:
  source_weight: 0.0
  include_digital_only_tracks: true
  search_max: 3
  art: false
  comments_separator: "\n---\n"
  exclude_extra_fields:
    - lyrics
    - comments
    - genre
    - style
    - day
    - month
    - medium
    - mediums
    - medium_index
    - albumstatus
    - albumtypes
  genre:
    capitalize: no
    maximum: 1
    always_include: []
    mode: classical # classical, progressive or psychedelic

#not using given conflict w/ inline albumartist_adj
the:
  a: yes
  the: yes
  patterns: ^th ^a ^an
  strip: no
  format: '{0},{1}'

#inline
item_fields:
  catalognum_adj: catalognum.replace(' ', '-') #.replace('-', '_')
album_fields:
  albumartist_adj: |
    import re
    return re.sub(r'([a-z])\1+', r'\1', re.sub('(?<=\w)[aeiou ]', '',
albumartist, flags=re.I))
  initials: |
    initials = ''
    for word in albumartist.replace('-', ' ').split():  #go through each
word
      initials += word[0].upper()  #append the initial
    return initials
  longAA: 1 if
    (len(albumartist) - albumartist.count(' ')) >= 16 else 0
  shortAA: 1 if
    (len(albumartist) - albumartist.count(' ')) <= 4 else 0
  catalognum_adj: catalognum.replace(' ', '-')
  andAA: 1 if '&' in albumartist else 0
  and_adj: |
    import re
    names = []
    for name in albumartist.split('&'):
      if (len(name) - name.count(' ') - len(re.findall('[()!#.?,~]',
name))) >= 16:
        initials = ''
        for word in name.replace('-', ' ').split():  #go through each word
          initials += word[0].upper()  #append the initial
        names.append(initials)
      elif (len(name) - name.count(' ')) <= 4:
        names.append(name.replace(' ', ''))
      else:
       #import re > from before initial import
        names.append(re.sub(r'([a-z])\1+', r'\1', re.sub('(?<=\w)[aeiou ]',
'', name, flags=re.I)))
    return '&'.join(names).replace(' ', '')

ftintitle:
  format: ft. {0}

directory: /Users/bresnick/Downloads/MusicTest

import:
  write: yes
  copy: yes
  languages: en
  autotag: no

art_filename: ${catalognum_adj}_art

#va_name: %if{albumtype='mix',$arranger,$label} not used bc references
mediabrainz not file

paths:
paths:
  #default:
%if{$compilation,Music/Compilations/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}},Music/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}}}
  #default:
Music/Compilations/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}}
  ###default:
Music/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}}
  ###comp:
Music/Compilations/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}}
  ##default:
Music/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}}
  ##comp:
Music/Compilations/%if{$longAA,$initials${year}_$catalognum_adj/${catalognum_adj}_$track,%if{$shortAA,$albumartist${year}_$catalognum_adj/${catalognum_adj}_$track,$albumartist_adj${year}_$catalognum_adj/${catalognum_adj}_$track}}
  default: Music/$and_adj${year}_$catalognum_adj/${catalognum_adj}_$track
  comp:
Music/Compilations/$and_adj${year}_$catalognum_adj/${catalognum_adj}_$track

ui:
  color: yes
  colors:
    text_success: green
    text_warning: yellow
    text_error: red
    text_highlight: red
    text_highlight_minor: lightgray
    action_default: turquoise
    action: blue

**Reply by DjSlash** [reply, marked as answer]
For future reference: please put something like the above in codeblocks [using triple backticks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).

Regarding your problem, here it is:
> `autotag: no`

With this set to "no", beets will [not try to tag your files](https://beets.readthedocs.io/en/stable/reference/config.html#autotag). Therefore it will not query discogs nor will it output anything about it.

**Comment by bkresnick** [comment]
Changing that setting did not resolve the issue.

Best regards,
Ben Resnick
+1 (720) 729-7450
